truckersmp_cli/__init__.py

```diff
@@ -886,6 +933,12 @@ def create_arg_parser():
       help="""start singleplayer game, useful for save editing,
               using/testing DXVK in singleplayer, etc.""",
       action="store_true")
+    ap.add_argument(
+      "--self-update",
+      help="""update files to the latest release.
+              Note: Python package users should use
+              "pip install truckersmp-cli --upgrade" instead""",
```

Comment:
I would get rid of the exact example because we don't know the installation arguments. Let's only tell them to use pip.